src/components/i-calendar.vue

```diff
@@ -0,0 +1,537 @@
+<template>
+  <div class="i-calendar">
+    <div class="i-calendar--header">
+      <div class="i-calendar--month-year">
+        <button
+          class="i-calendar--btn-chevron"
+          :class="{ disabled: disabledPreviousMonth }"
+          :disabled="disabledPreviousMonth"
+          @click="clickPreviousMonth"
+        >
+          <ic-angle-circle
+            direction="left"
+            class="icon-angle-large"
+          />
+        </button>
+        <button
+          class="btn-active-month-year"
+          @click="clickMonthPicker"
+        >
+          <span class="active-month">{{ activeMonth }}</span> <span>{{ activeYear }}</span>
+        </button>
+        <button
+          class="i-calendar--btn-chevron"
+          :class="{ disabled: disabledNextMonth }"
+          :disabled="disabledNextMonth"
+          @click="clickNextMonth"
+        >
+          <ic-angle-circle
+            direction="right"
+            class="icon-angle-large"
+          />
+        </button>
+      </div>
+      <div class="indicator-color">
+        <slot name="indicator" />
+      </div>
+      <div
+        v-if="showMonthYearTable"
+        class="floating-month-year-pop-up"
+      >
+        <div class="year-wrapper">
+          <button
+            class="i-calendar--btn-chevron"
+            :class="{ disabled: disabledPreviousMonth }"
+            :disabled="disabledPreviousMonth"
+            @click="clickPreviousYear"
+          >
+            <ic-angle-circle direction="left" />
+          </button>
+          <div class="active-year">
+            <span>{{ activeYearPickerString }}</span>
+          </div>
+          <button
+            class="i-calendar--btn-chevron"
+            :class="{ disabled: disabledNextMonth }"
+            :disabled="disabledNextMonth"
+            @click="clickNextYear"
+          >
+            <ic-angle-circle direction="right" />
+          </button>
+        </div>
+        <div class="month-wrapper">
+          <div
+            v-for="(month, key) in listMonth"
+            :key="`month-${key}`"
+            class="list-month"
+          >
+            <button
+              class="btn-month"
+              :class="{ selected: isSelectedMonth(month.monthValue) }"
+              @click="clickMonth(month)"
+            >
+              <div class="btn-month--label">
+                {{ month.monthDisplay }}
+              </div>
+              <div
+                v-if="isThereNewUpdate"
```

Comment:
isThereNewUpdate should be not a props (you can ask me for the details)

Reply:
ok